ui/config/data/general_config.js

```diff
@@ -41,5 +41,76 @@ UserConfig.registerOptions('general', {
       type: 'checkbox',
       default: false,
     },
+    {
+      id: 'DisplayLanguage',
+      name: {
+        en: 'Display language',
+        ko: '주 사용 언어',
+      },
+      type: 'select',
+      options: {
+        en: {
+          'Use FFXIV Plugin Language': 'default',
+          'English (en)': 'en',
+          'Chinese (cn)': 'cn',
+          'German (de)': 'de',
+          'French (fr)': 'fr',
+          'Japanese (ja)': 'ja',
+          'Korean (ko)': 'ko',
+        },
+        de: {
+          'Benutze FFXIV Plugin Sprache': 'default',
+          'Englisch (en)': 'en',
+          'Chinesisch (cn)': 'cn',
+          'Deutsch (de)': 'de',
+          'Französisch (fr)': 'fr',
+          'Japanisch (ja)': 'ja',
+          'Koreanisch (ko)': 'ko',
+        },
+        fr: {
+          'Utiliser la langue du Plugin FFXIV': 'default',
```

Comment:
Just correction default in french, here :
`'Utiliser la langue du Plugin FFXIV' : 'par défaut',`

Reply:
latter is not localized I think, it's used as common thing.
like `if(language == "ko")` thing?

Reply:
I did not understand.
If you don't think it's useful to translate it, don't do it. I would look on the software after its update.

Reply:
The values on the right hand side here are not shown to users and is only used by code.  The only valid values are 'default' or something like 'en' or 'fr' etc.  They should not be translated or the code will break.

Reply:
yes, that's what I was to say. my bad English :(

Reply:
Ah no, I understood what you meant.  I was trying to say it differently in the hopes that our comments combined would communicate clearly here.  :)